ovos_utils/system.py

```diff
@@ -77,6 +77,8 @@ def ssh_disable():
     subprocess.call('sudo systemctl stop ssh.service', shell=True)
     subprocess.call('sudo systemctl disable ssh.service', shell=True)
 
+def mycroft_restart():
```

Comment:
can this be renamed to clarify that it only restarts system services? `restart_mycroft_service`

otherwise people may expect this to work like the start-mycroft.sh script

Reply:
done, and also changed it in the place its being called https://github.com/OpenVoiceOS/ovos-PHAL-plugin-system/pull/2